Example/SmileID.xcodeproj/project.pbxproj

```diff
@@ -918,6 +922,7 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				DEVELOPMENT_TEAM = "";
 				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = 99P7YGX9Q6;
+				ENABLE_SENTRY = NO;
```

Comment:
@JNdhlovu I noticed that `ENABLE_SENTRY` is enabled for Debug but disabled for Release. Is there a reason why `ENABLE_SENTRY` is set to `NO` for Release?